docs/docsite/rst/porting_guides/porting_guide_2.5.rst

```diff
@@ -75,6 +75,22 @@ Included file:
 
 The relevant change in those examples is, that in Ansible 2.5, the included file defines the tag ``distro_include`` again. The tag is not inherited automatically.
 
+Fixed handling of keywords and inline variables
+-----------------------------------------------
+
+Several fixes to how we handle keywords and 'inline variables' help avoid conflating ones with the others, sadly this has broken a common usage of 'name', as it always was a reserved keyword but freely used as a variable when calling roles::
```

Comment:
In 2.5 we made several fixes to how we handle keywords and 'inline variables', to avoid conflating the two. Unfortunately these changes mean you must specify whether `name` is a keyword or a variable when calling roles. If you have playbooks that look like this: